composer.json

```diff
@@ -21,8 +22,17 @@
       "OpenEuropa\\CodeReview\\Tests\\": "./tests/"
     }
   },
+  "extra": {
+    "enable-patching": true,
+    "composer-exit-on-patch-failure": 1,
+    "patches": {
+      "squizlabs/php_codesniffer": {
+        "Fix conflict between scope indentation and array indentation checking. Ref. https://github.com/squizlabs/PHP_CodeSniffer/issues/2281": "./resources/patches/phpcs-390bffe2265aac8184fdf86400c0d0841e65bc4b.patch",
+        "Disable exact checking of multi-line chained method calls in Generic.WhiteSpace.ScopeIndent. Ref. https://github.com/squizlabs/PHP_CodeSniffer/pull/2372": "./resources/patches/phpcs-pr-2372.patch"
```

Comment:
Can we use the link of the patch?
`https://github.com/squizlabs/PHP_CodeSniffer/commit/6e315f1945c4e97aa183b246e014d93de027b8e2.patch`

Reply:
Vitor, if you apply the first patch, you won't be able to apply the next one.
Both local patches are different from what you can find on their corresponding issues.

Reply:
These patches must stay local for now until something is moving in squizlabs/php_codesniffer

Reply:
@pfrenssen @drupol We should put these patches online, because if we try to use this branch in our other project, this issue appears:
```
  - Applying patches for squizlabs/php_codesniffer
    ./resources/patches/phpcs-390bffe2265aac8184fdf86400c0d0841e65bc4b.patch (Fix conflict between scope indentation and array indentation checking. Ref. https://github.com/squizlabs/PHP_CodeSniffer/issues/2281)
   Could not apply patch! Skipping. The error was: The "./resources/patches/phpcs-390bffe2265aac8184fdf86400c0d0841e65bc4b.patch" file could not be downloaded: failed to open stream: No such file or directory
```
I try to find a conversation on ` squizlabs/PHP_CodeSniffer` talking concerning this issue.

Reply:
I found the issue https://github.com/cweagans/composer-patches/pull/161 and the PR https://github.com/cweagans/composer-patches/pull/218

Reply:
Nice nice

Reply:
So I have two questions:
- @pfrenssen Why we are using custom patches stored on local and their should be in an issue queue? Like that, we are loosing the process of the patch.
- If we want to use local files, could we block this PR to work on or wait a new version of ```cweagans/composer-patches``` (https://github.com/cweagans/composer-patches/pull/218)

Reply:
The PR218 seems to be abandonned... I created a new one and updated it accordingly.

https://github.com/cweagans/composer-patches/pull/247

I would suggest to not lose too much time on this as this is, according to me, out of scope.